src/components/HomePage.tsx

```diff
@@ -0,0 +1,3 @@
+export const HomePage: React.FC = () => (
+  <h1 className="title">Home page</h1>
+);
```

Comment:
Let's use the container div to wrap this content

```suggestion
export const HomePage: React.FC = () => (
  <div className="container">
    <h1 className="title">Home page</h1>
  </div>
);
```